index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once 'Autor.php';
+require_once 'Libro.php';
+require_once 'Revista.php';
+require_once 'ImprimirAutor.php';
+
+$autor1 = new Autor("Elena", "White", 45);
```

Comment:
```suggestion
$autorElenaWhite = new Autor("Elena", "White", 45);
```
El nombre `$autor1` fue agregado a modo de ilustración. No olvide que los nombres de las variables deben ser descriptivas.